backend/sXb-service/Services/EmailSender.cs

```diff
@@ -1,71 +1,89 @@
-﻿using System.Net;
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Net.Mail;
+using System.Text;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Options;
+using RestSharp;
+using RestSharp.Authenticators;
 using sXb_service.Helpers;
 
-namespace sXb_service.Services
-{
+namespace sXb_service.Services {
     // This class is used by the application to send email for account confirmation and password reset.
     // For more details see https://go.microsoft.com/fwlink/?LinkID=532713
-    public class EmailSender : IEmailSender
-    {
+    public class EmailSender : IEmailSender {
         public IConfiguration Configuration { get; }
         private readonly SMTPConfig smtpConfig;
-        public EmailSender(IOptions<AuthMessageSenderOptions> optionsAccessor, SMTPConfig smtpConfig)
-        {
+        public EmailSender (IOptions<AuthMessageSenderOptions> optionsAccessor, SMTPConfig smtpConfig) {
             this.smtpConfig = smtpConfig;
             Options = optionsAccessor.Value;
         }
 
         public AuthMessageSenderOptions Options { get; } //set only via Secret Manager
 
-        public void SendEmailAsync(string email, string subject, string message)
+        public void SendEmailAsync (string email, string subject, string message) {
+            //Execute (subject, message, email);
```

Comment:
**This breaks trading.** 
If were going to use mailgun then it has to be passed in an email, subject, and message. 
Since mailgun still has delivery problems within WVUP, we probably shouldn't even use it.